sdk/engine/source/graphics/techniques/WireframeTechnique.cpp

```diff
@@ -23,7 +23,6 @@ namespace nc::graphics
       m_pipelineLayout{nullptr}
     {
         CreatePipeline(renderPass);
```

Comment:
Removed because in previous version not using signals, the graphics module was constructed before the assets utility in the runtime. It was causing problems trying to load a mesh asset in the ctor here.
WireframeTechnique does not need it here anyway